### PR TITLE
File 489

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: docs/file-upload-process.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+## FILE UPLOAD PROCESS
+Below describes how the “file-upload” process works. The process described below is recommended if the client aims to upload multiple file (clients that aim to only upload a single file and are not interested in routing can follow a different [process](#single-file-process)): -
+
+### LIFE-CYCLE OF AN ENVELOPE - [see state diagram](../resources/images/envelope-life-cycle.png)
+
+#### 1. Create an envelope
+The first step is to create an envelope. The following endpoint is called and the envelope is created (an envelopeId is generated and returned in the responseHeader) : -
+
+- POST       /file-upload/envelopes
```
Comment (YuanZHANG910) [inline docs/file-upload-process.md]: You might need to point out that some calls such as this call it is a backend call : {{be-url}}/file-upload/envelopes

File: docs/file-upload-process.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+#### Envelope Statuses
+See [here](https://github.com/hmrc/file-upload#envelope-statuses) for envelope statuses
+
+### SINGLE FILE PROCESS
+Clients that aim to upload a single file and have no need for "routing", can follow the following steps: -
+1. [create an envelope](https://github.com/hmrc/file-upload#create-an-envelope)
+2. [upload file](https://github.com/hmrc/file-upload-frontend#upload-file)
+3. [download file](https://github.com/hmrc/file-upload#download-file)
+
```
Comment (YuanZHANG910) [inline docs/file-upload-process.md]: Files can be deleted as well: {{be-url}}/envelopes/{{envelopeId}}/files/{{fileId}}

Comment (habeeb01) [reply docs/file-upload-process.md]: good point, will add that

**Comment by kastoestoramadus** [comment]
Hi, This a repository for frontend part. You describe full service. I think it is not best place for that.
Frontend is only for the one step. Backend ows the model. Either we put it in confluence or in backend repo.

**Pull request merged**